.github/workflows/docker-hub.yml

```diff
@@ -0,0 +1,47 @@
+# Ref: https://github.com/docker/build-push-action#usage
+name: Build Docker container
+
+on:
+  push:
+    branches:
+      - development
+      - feature/gh-action-build-docker
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Set Docker Hub namespace
+        if: ${{ github.repository_owner == 'fossasia' }}
+        run: echo "DOCKERHUB_SPACE=eventyay" >> $GITHUB_ENV
+      -
+        name: Set Docker Hub namespace
+        if: ${{ github.repository_owner != 'fossasia' }}
+        run: echo "DOCKERHUB_SPACE=${{ github.repository_owner }}" >> $GITHUB_ENV
```

Comment:
I'm not an expert on this so could you clarify why we need these checks?

Reply:
Because this code is still on my repo. If I add some more commits, the script still runs on my repo and it needs a destination to push image to. So I let it push to my personal Docker Hub in case the code is not merged yet.

Reply:
ok, let's merge and test then we can change it later